spi/gnap/internal/jwksignature/ecdsa.go

```diff
@@ -0,0 +1,113 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package jwksignature
+
+import (
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/rand"
+	"encoding/asn1"
+	"errors"
+	"fmt"
+	"math/big"
+
+	"github.com/hyperledger/aries-framework-go/pkg/doc/jose/jwk"
+)
+
+// copied from afgo.
```

Comment:
source? this will help track changes (if needed)

Reply:
added